sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -182,19 +182,19 @@ foreach (ConversationEntity entity in conversationPrediction.Entities)
 }
 ```
 
-Additional options can be passed to `AnalyzeConversations` like enabling more verbose output:
+Additional options can be passed to `AnalyzeConversation` like enabling more verbose output:
```

Comment:
Minor question. I'm assuming you meant this to stay AnalyzeConversation and not CLU right?

Reply:
The API does not change with CLU